app/src/main/java/memphis/myapplication/data/RealmRepository.java

```diff
@@ -214,6 +214,7 @@ public SecretKey getSymKey(String friendName) {
             User user = userRealmToUser(realm.where(UserRealm.class).equalTo("username", friendName).findFirst());
             byte[] encryptedKey = user.getSymKey();
             realm.commitTransaction();
+            if (encryptedKey == null) return null;
```

Comment:
Usually discourage these types of formatting in ndn cxx code style. Not sure about Java/jndn.
https://named-data.net/doc/ndn-cxx/current/code-style.html